utils/clflags.ml

```diff
@@ -577,3 +577,24 @@ let create_usage_msg program =
 
 let print_arguments program =
   Arg.usage !arg_spec (create_usage_msg program)
+
+let auto_include_otherlibs otherlibs alert find_in_dir fn =
+  if !no_std_include then raise Not_found
+  else
+    let scan (lib, dir) =
+      let dir = Lazy.force dir in
```

Comment:
```suggestion
    let scan (lib, lazy dir) =
```

Reply:
I wish I could say I knew we could do that 🙈